lib/fluent/plugin/out_azure-storage-append-blob.rb

```diff
@@ -56,6 +57,10 @@ def configure(conf)
         if @azure_container.nil?
           raise ConfigError, 'azure_container is needed'
         end
+
+        if @azure_storage_access_key.nil? || @azure_storage_sas_token.nil?
```

Comment:
I think you want to be using `&&` instead of `||`, right?

Reply:
`&&` is "logical AND" operator. The idea was to check that at least one of provided params is not empty, so it should be the "logical OR".

Reply:
Oh, damn it, @elsesiy you're right, thank you. Will fix that.

Reply:
Updated in 055bd6855b218bf0ad88cd8fbac0fe8926324972

Reply:
You’re welcome! But it would be good to know how you made this work regardless, see my other comment :)